### DISCUSSION TITLE
Como Incentivar um Homem a Fazer Exercício

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#492] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/como-incentivar-um-homem-a-fazer-exercicio/

**Comment by angellisy** [comment]
Hmmm... :smiling\_imp:

**Comment by one-eyed-willy** [comment]
> [angellisy](</perfil/?user=angellisy>) disse [segunda-feira, 4 de julho de 2022](<https://www.vish.blog.br/videos/como-incentivar-um-homem-a-fazer-exercicio/#comment_1173950314>):

> Hmmm... 😈

![null](<https://i.imgur.com/m26Mc7i.gif>)
